validator/main.py

```diff
@@ -39,14 +39,13 @@ class LlamaGuard7B(Validator):
     def __init__(
         self,
         policies: Optional[List[str]] = None,
-        validation_method: Optional[str] = "full",
```

Comment:
i dont see this used anywhere in our core oss codebase?

Reply:
it's in some validators for determining sentence level vs full context level validation. If it's not used in this exact code, it's not necesasry